DECLARING_DEPENDENCIES.md

```diff
@@ -0,0 +1,209 @@
+# Declaring Dependencies on Google Java libraries
+
+Google maintains a number of open source Java libraries that make it
+easier to use services in Google Cloud Platform. Additionally Google
```

Comment:
```suggestion
easier to use services in Google Cloud Platform (GCP). Additionally Google
```

To refer it with the abbreviation in other places.

Reply:
Will do.